README.md

```diff
@@ -20,7 +20,7 @@ Then upload the image to the desired repository.
 
 To run the prometheus exporter:
 
-    bundle exec bin/prometheus-metadata-exporter -m METADATA_URL --cas DIRECTORY_OF_CA_CERTIFICATE_FILES [--allow_self-signed]
+    bundle exec bin/prometheus-metadata-exporter -m METADATA_URL --cas DIRECTORY_OF_CA_CERTIFICATE_FILES [--allow_self_signed]
```

Comment:
Is this our binary? Standard practice would be to use dashes rather than underscores, if we have any say in the matter :-)

Reply:
Yes it's ours - thanks. I'll change it asap.